src/mame/drivers/hp80.cpp

```diff
@@ -120,6 +122,7 @@ class hp85_state : public driver_device
 	required_ioport m_io_key1;
 	required_ioport m_io_key2;
 	required_ioport m_io_modkeys;
+	required_device<hp80_optrom_slot_device> m_rom_drawers[ 6 ];
```

Comment:
Can you use `required_device_array<hp80_optrom_slot_device, 6>` instead here?  The initialisation will look prettier then: `m_rom_drawers(*this, "drawer%u", 1)` (parameters are device, tag format, and starting number for the `%u` in the tag format).